README.md

```diff
@@ -12,3 +12,83 @@ Playbook for deploying OSBS into [OpenShift Dedicated](https://www.openshift.com
 
 4. Run `deploy.sh`. Any arguments will be passed to `ansible-playbook` - you
    might want to use e.g. `--verbose`.
+
+
+Requirements for self-hosted OSE instance
+=========================================
+
+A bare self-hosted OSE installation should be sufficient for OSBS to work,
+as it doesn't require routers, docker registry or SDN to function.
+
+
+Role Variables
+==============
+
+Specify project namespace
+```
+osbs_namespace: best-namespace
+```
+
+Create service accounts. 
+
+Depending on your configuration OSBS needs 
+service accounts to communicate with koji and pulp (see osbs-client configuration).
```

Comment:
Aren't service accounts more about allowing e.g. koji to communicate with OSBS? It's just an account that a service (e.g. koji) can use to authenticate itself with OpenShift.

I don't think Pulp has a service account, as it does not need to make any requests to OpenShift.